src/store/modules/product/actions.ts

```diff
@@ -42,6 +42,23 @@ const actions: ActionTree<ProductState, RootState> = {
     return resp;
   },
 
+  async fetchProduct ({ commit }, { productId }) {
+    const resp = await ProductService.fetchProducts({
+      "filters": ['productId: ' + productId ]
+    })
+    if (resp.status === 200 && !hasError(resp)) {
+      const product = resp.data.response.docs[0];
+      if (resp.data) commit(types.PRODUCT_ADD_TO_CACHED, { product });
+    }
+    // TODO Handle specific error
+    return resp;
+  },
+
+  async updateCurrent({ commit }, { product }) {
+    commit(types.PRODUCT_CURRENT_UPDATED, { product })
```

Comment:
Directly pass product as param instead of passing it as object.

Reply:
Updated at all places